src/internal/operations/manifests.go

```diff
@@ -151,18 +150,33 @@ func produceManifestsAndMetadata(
 			return ms, nil, false, nil
 		}
 
-		colls, err := collectMetadata(mctx, mr, man, metadataFiles, tenantID, errs)
+		// a local fault.Bus intance is used to collect metadata files here.
+		// we avoid the global fault.Bus because all failures here are ignorable,
+		// and cascading errors up to the operation can cause a conflict that forces
+		// the operation into a failure state unnecessarily.
+		// TODO(keepers): this is not a pattern we want to
+		// spread around.  Need to find more idiomatic handling.
+		fb := fault.New(true)
+
+		colls, err := collectMetadata(mctx, mr, man, metadataFiles, tenantID, fb)
 		if err != nil && !errors.Is(err, data.ErrNotFound) {
+			LogFaultErrors(ctx, fb.Errors(), "collecting metadata")
```

Comment:
move this above the if-block and remove the one lower down?

Reply:
Sure, why not